current_spring2026/evaluation/logger.py

```
@@ -0,0 +1,126 @@
+                        intent.raw_query,
+                        intent.rewritten_query,
+                        intent.query_type,
+                        filters_json,
+                        retrieved_arks,
+                        generation_result.response,
+                        relevancy_score,
+                        faithfulness_score,
+                        latency_ms,
+                    ),
+                )
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)